examples/decoding/plot_haxby_full_analysis.py

```diff
@@ -23,10 +23,12 @@
 
 # load labels
 import numpy as np
-labels = np.recfromcsv(haxby_dataset.session_target[0], delimiter=" ")
+labels = np.recfromcsv(haxby_dataset.session_target[0], delimiter=" ",
+                       dtype=[('labels', 'U12'), ('chunks', '<i8')])
```

Comment:
Thanks for catching this.  I think this was my attempted solution for examples, before simply prefixing all strings as byte-strings.  I've removed now.